benchmarks/api/fabric/couchDB/contract/create-asset.yaml

```diff
@@ -34,26 +36,29 @@ test:
     callback: benchmarks/api/fabric/lib/create-asset.js
   - label: create-asset-4000
     description: Test a submitTransaction() Gateway method against the NodeJS `fixed-asset` Smart Contract method named `createAsset`, which inserts an asset of size 4000 bytes into the World State database.
-    chaincodeId: fixed-asset
+    chaincodeID: fixed-asset
     txDuration: 300
     rateControl: { type: fixed-backlog,  opts: { unfinished_per_client: 10, startingTps: 1} }
     arguments:
+      chaincodeID: fixed-asset
       bytesize: 4000
     callback: benchmarks/api/fabric/lib/create-asset.js
   - label: create-asset-8000
     description: Test a submitTransaction() Gateway method against the NodeJS `fixed-asset` Smart Contract method named `createAsset`, which inserts an asset of size 8000 bytes into the World State database.
-    chaincodeId: fixed-asset
+    chaincodeID: fixed-asset
     txDuration: 300
     rateControl: { type: fixed-backlog,  opts: { unfinished_per_client: 10, startingTps: 1} }
     arguments:
+      chaincodeID: fixed-asset
       bytesize: 8000
     callback: benchmarks/api/fabric/lib/create-asset.js
   - label: create-asset-16000
     description: Test a submitTransaction() Gateway method against the NodeJS `fixed-asset` Smart Contract method named `createAsset`, which inserts an asset of size 16000 bytes into the World State database.
-    chaincodeId: fixed-asset
+    chaincodeID: fixed-asset
     txDuration: 300
     rateControl: { type: fixed-backlog,  opts: { unfinished_per_client: 10, startingTps: 1} }
     arguments:
+      chaincodeID: fixed-asset
       bytesize: 16000
     callback: benchmarks/api/fabric/lib/create-asset.js
   - label: create-asset-24000
```

Comment:
Need to change `chaincodeId` to `chaincodeID` for the create-asset-24000 round and also add `chaincodeID: fixed-asset` to the arguments